its/src/test/java/com/sonar/it/csharp/IncrementalAnalysisTest.java

```diff
@@ -163,23 +185,40 @@ private void addIssue(File file) throws IOException {
   }
 
   private void createFileWithIssue(File file) throws IOException {
+    createFileWithContent(file,
+      "namespace IncrementalPRAnalysis\n" +
+        "{\n" +
+        "public class AddedFile\n" +
+        "{\n" +
+        "}\n" +
+        "}// FIXME: S1134");
+  }
+
+  private void createDuplicate(File oldFile, File newFile) throws IOException {
+    BufferedReader reader = new BufferedReader(new FileReader(oldFile));
+    String content = "";
+    String currentLine;
+    while ((currentLine = reader.readLine()) != null) {
+      content += currentLine + System.lineSeparator();
+    }
+    reader.close();
+    content = content.replace("OriginalClass", "CopyClass");
+    createFileWithContent(newFile, content);
```

Comment:
With this change it started failing on the CI with the following error:
`\CopyClass.cs(1,1): error CS1031: Type expected`
I have reverted it so it can be merged, I will investigate the problem once I am back.